include/zephyr/bluetooth/mesh/msg.h

```diff
@@ -122,6 +122,7 @@ struct bt_mesh_msg_ctx {
 		.app_idx = (app_key_idx), \
 		.addr = (dst), \
 		.send_ttl = (ttl), \
+		.rnd_delay = false \
```

Comment:
This is not needed. All not-specified fields in the designated initializer will be set to 0 by standard.